src/runtime_src/core/tools/common/XBHelpMenus.cpp

```diff
@@ -532,20 +532,40 @@ XBUtilities::create_suboption_list_string(const VectorPairStrings &_collection)
   // Working variables
   const unsigned int maxColumnWidth = m_maxColumnWidth - m_shortDescriptionColumn; 
   std::string supportedValues;        // Formatted string of supported values
-                                      
-  // Make a copy of the data (since it is going to be modified)
-  VectorPairStrings workingCollection = _collection;
+
+  // Map seconds to their firsts
+  // Allows for multiple firsts to be linked to a single second
+  std::map<std::string, std::vector<std::string>> descriptionToNames;
+  for(const auto & pairs : _collection)
+  {
+    descriptionToNames.emplace(pairs.second, std::vector<std::string>());
+    descriptionToNames[pairs.second].push_back(pairs.first);
+  }
 
   // Determine the indention width
+  // Create working collection
+  VectorPairStrings workingCollection;
   unsigned int maxStringLength = 0;
-  for (auto & pairs : workingCollection) {
-    // Determine if the keyName needs to have 'quotes', if so add them
-    if (pairs.first.find(' ') != std::string::npos ) {
-      pairs.first.insert(0, 1, '\'');  
-      pairs.first += "\'";     
+  for(auto & keyValuePair : descriptionToNames) {
+    // The string that will contain all names when finished
+    std::string currentStringName = "";
+    // Go through the vector of possible names to assemble the suggestions for the menu
+    for (long unsigned int i = 0; i < keyValuePair.second.size(); i++) {

```

Comment:
It would be preferred to use size_t instead.
```suggestion
    for (size_t i = 0; i < keyValuePair.second.size(); i++) {

```